src/scss/app.scss

```diff
@@ -1,13 +1,137 @@
+@import "media";
+@import "typography";
+@import "components/normalise";
 
-$primary-color: green;
+$background-primary: #36393f;
+$background-secondary: #2f3136;
+$background-secondary-alt: #292b2f;
+$background-tertiary: #202225;
+$background-accent: #4f545c;
+$background-floating: #18191c;
+$border-primary: rgba(4,4,5,0.5);
+$interact-hover: #dcddde;
+$interact-active: #fff;
+$interact-muted: #4f545c;
 
-h1 {
-    color: $primary-color;
+.app {
+    background-color: $background-tertiary;
+    height: 100vh;
+    width: 100%;
+    display: flex;
+
+    @include breakpoint(sm) {
+        flex-direction: column;
+    }
+}
+
+.side {
+    background-color: $background-tertiary;
+    height: 100%;
+    display: flex;
+    flex: 0 0 auto;
+
+    @include breakpoint(sm) {
+        width: auto;
+        height: 500px;
+    }
+}
+
+.side-server {
+    width: 72px;
+    height: 100%;
+    padding: 8px;
+    flex: 0 0 auto;
+}
+
+.side-server-home {
+
+}
+
+.side-server-channel {
+
+}
+
+.side-server-download {
+
+}
+
+
+.side-message {
+    background-color: $background-secondary;
+    width: 240px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    flex: 1 0 auto;
+
+    @include breakpoint(md) {
+        width: 72px;
+    }
+
+    @include breakpoint(sm) {
+        width: auto;
+        font-size: 1rem;
+    }
 }
 
-p{
-    width:50px;
-    height:50px;
-    background-image: url("../img/diamond.svg");
-    background-size: contain;
-}
\ No newline at end of file
+.side-message-start {
+    height: 48px;
+    padding: 0 10px;
+    border-bottom: 1px solid $border-primary;
```

Comment:
For `1px` things, I'm generally ok with denoting `1px` it's more informative that `0.1rem`